modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is currently offered as an alternative to Elasticsearch as a log store for the {logging}.
```

Comment:
This could be a bit better? Loki is a horizontally scalable, highly available, multi-tenant log aggregation system and is currently planned to be offered as an alternative to Elasticsearch as a log store for the logging subsystem

WDYT?

Reply:
Gotcha. I mentioned that considering we still haven't GA'd it yet.